addons/project_todo/static/src/components/todo_done_checkmark/todo_done_checkmark.js

```diff
@@ -0,0 +1,75 @@
+/** @odoo-module */
+
+const { useState, onRendered, onMounted } = owl;
```

Comment:
We should use the new import for that instead of using the global variable 🙂 

```suggestion
import { useState, onRendered, onMounted } from "@odoo/owl";
```